Content.Server/Motd/MOTDCommand.cs

```diff
@@ -0,0 +1,66 @@
+using Content.Server.Administration;
+using Content.Server.Administration.Logs;
+using Content.Server.Administration.Managers;
+using Content.Server.Chat.Managers;
+using Content.Server.Chat.Systems;
+using Content.Shared.Administration;
+using Content.Shared.CCVar;
+using Content.Shared.Chat;
+using Content.Server.Commands;
+using Content.Shared.Database;
+using Robust.Server.Player;
+using Robust.Shared.Configuration;
+using Robust.Shared.Console;
+
+namespace Content.Server.Motd;
+
+/// <summary>
+/// A console command which either prints the current message of the day or sets the current message of the day.
+/// Setting the message of the day is a privileged action that can only be performed by the server host or admins with the <see cref="AdminFlag.Admin"/> flag.
+/// Printing the message of the day, on the other hand, is an unprivileged action any player may perform.
+/// </summary>
+[AnyCommand]
+internal sealed class MOTDCommand : IConsoleCommand
+{
+    #region Dependencies
+    [Dependency] private readonly IAdminLogManager _adminLogManager = default!;
+    [Dependency] private readonly IAdminManager _adminManager = default!;
+    [Dependency] private readonly IChatManager _chatManager = default!;
+    [Dependency] private readonly IConfigurationManager _configurationManager = default!;
+    [Dependency] private readonly IEntitySystemManager _entitySystemManager = default!;
+    private MOTDSystem? _motdSystem = null; // Can't be a normal dependency because only entity systems can resolve entity systems as dependencies.
+    #endregion Dependencies
+
+    public string Command => "motd";
+    public string Description => "Print or set the Message Of The Day.";
+    public string Help => "motd [ <text> ]";
+    
+    public void Execute(IConsoleShell shell, string argStr, string[] args)
+    {
+        var player = (IPlayerSession?) shell.Player;
+        if (args.Length < 1)
+        {   // Get MOTD
+            _motdSystem ??= _entitySystemManager.GetEntitySystem<MOTDSystem>();
```

Comment:
You dont need to set the dependency, just resolve it here
```suggestion
            _motdSystem = _entityManager.System<MOTDSystem>();
```

Reply:
Done